src/components/Personalization/index.js

```diff
@@ -87,7 +87,7 @@ const createCollect = eventManager => {
 
     mergeMeta(event, meta);
 
-    eventManager.sendEvent(event);
+    // eventManager.sendEvent(event);
```

Comment:
Did you mean to leave this commented out?

Reply:
yes, otherwise we'll enter a loop. I'll fix this in next PR.